javascript/statements.json

```diff
@@ -1864,7 +1864,8 @@
                 }
               ],
               "firefox": {
-                "version_added": false
+                "version_added": false,
+                "notes": "See <a href='https://bugzilla.mozilla.org/show_bug.cgi?id=1247687'>bug 1247687</a>"
               },
               "firefox_android": {
                 "version_added": false
```

Comment:
I expect this also applies to Firefox for Android too. Would you mind duplicating the note here?